release_notes.md

```diff
@@ -1,4 +1,6 @@
 ### Release notes
 <!-- Please add your release notes in the following format:
 - My change description (#PR/#issue)
--->
\ No newline at end of file
+-->
+- Fixes `null` reference exception when accessing request identities (#673)
+- Exposed `RetryContext` to support retry policies (#316)
```

Comment:
nit: I see the first entry links the PR while the second one links the issue.

Reply:
Primarily because there was no issue tracking that change. For planned changes, with an associated issue, linking to the issue provides the context captured there and a link to the associated PR, but it's typically easier to start from the issue that drove the change.